CHANGELOG.md

```diff
@@ -17,8 +17,10 @@ and this project adheres to
 
 - cosmwasm-std: Enable `add_event` and `add_events` functions to process types
   implementing `Into<Event>` ([#2044])
+- cosmwasm-vm: Improve performance of the `Cache::analyze` function ([#2051])
```

Comment:
That was the other branch ;)

Reply:
Yep, whoops. Force pushed back to the previous state